web-components/src/components/header/index.tsx

```diff
@@ -208,53 +209,74 @@ export default function Header({
   // );
 
   const styles = useStyles({ color, borderBottom, fullWidth });
+  const showBanner = () => {
+    // if we're in the registry, where we have REACT_APP prefixed
+    // keys in the env vars, do not show the banner. in other words,
+    // only show the banner on the website.
+    return Object.keys(process.env).reduce((prev, curr) => {
+      if (curr.startsWith('REACT_APP')) {
+        return false;
+      } else {
+        return prev;
+      }
+    }, true);
+  };
   return (
-    <div
-      className={cx(
-        styles.borderBottom,
-        absolute && styles.absolute,
-        transparent ? styles.transparent : styles.background,
-      )}
-    >
-      <Container
-        disableGutters
-        className={styles.container}
-        maxWidth={fullWidth ? false : 'xl'}
+    <>
+      {showBanner() && <MarketplaceLaunchBanner />}
+      <div
+        className={cx(
+          styles.borderBottom,
+          absolute && styles.absolute,
+          transparent ? styles.transparent : styles.background,
+        )}
       >
-        <Box className={styles.header}>
-          <HomeLink
-            color={isTablet ? theme.palette.primary.contrastText : color}
-          />
-          <Box className={styles.desktop} display={{ xs: 'none', md: 'block' }}>
-            <MenuList className={styles.menuList}>
-              {menuItems?.map((item, index) => {
-                return (
-                  <HeaderMenuHover
-                    key={index}
-                    linkComponent={linkComponent}
-                    item={item}
-                    pathname={pathname}
-                  />
-                );
-              })}
-              {isRegistry && extras}
-            </MenuList>
-          </Box>
-
-          <Box className={styles.mobile} display={{ xs: 'block', md: 'none' }}>
-            <MobileMenu
-              linkComponent={linkComponent}
-              isRegistry={isRegistry}
-              pathname={pathname}
-              menuItems={menuItems}
-              isAuthenticated={isAuthenticated}
-              onLogin={onLogin}
-              onLogout={onLogout}
-              onSignup={onSignup}
+        <Container
+          disableGutters
+          className={styles.container}
+          maxWidth={fullWidth ? false : 'xl'}
+        >
+          <Box className={styles.header}>
+            <HomeLink
+              color={isTablet ? theme.palette.primary.contrastText : color}
```

Comment:
I know this variable was already declared in the file but it's to avoid using client responsive check when we can the same with pure CSS.
Here it would be `sx={{color:[color, theme.palette.primary.contrastText, color]}}` to do the same.

Reply:
so you mean rather than `color` prop use the `sx` prop you've provided? i'm glad to make the change but TBH i don't understand what it's doing. are you saying we can also get rid of the `isTablet` variable?

Reply:
I just checked and the HomeLink component does not have the `sx` props, so leave it as it is for now it's ok.
But basically the syntax `sx={{color: [xsValue, smValue, mdValue, lgValue, xlValue ]}}` is a shorthand to apply responsive values. It's best to use this syntax whenever possible because it uses CSS media queries while using `useMediaQuery` hook which `isTablet` came from is done client-side with JS. On top of being less performant, it can cause issues with server-side rendering which we'll probably have at some point.

Reply:
i see, thanks for the extra info about the @flagrede